apps/condo/migrations/20211204030000-87_manual_set_dataFormat_null.js

```diff
@@ -0,0 +1,11 @@
+exports.up = async (knex) => {
+    await knex.raw(`
+        BEGIN;
+        UPDATE "BillingIntegration" SET "dataFormat" = null WHERE true;
```

Comment:
Do you want to set it for all `BillingIntegration`?

Reply:
yep